lib/content-services/src/lib/search/components/search-filter-chips/search-facet-chip-tabbed/search-facet-chip-tabbed.component.spec.ts

```diff
@@ -0,0 +1,232 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ContentTestingModule } from '../../../../testing/content.testing.module';
+import { TranslateModule } from '@ngx-translate/core';
+import { By } from '@angular/platform-browser';
+import { SearchQueryBuilderService } from '../../../services/search-query-builder.service';
+import { SearchFilterList } from '../../../models/search-filter-list.model';
+import { SearchFacetChipTabbedComponent } from './search-facet-chip-tabbed.component';
+import { TabbedFacetField } from '../../../models/tabbed-facet-field.interface';
+import { FacetField } from '../../../models/facet-field.interface';
+import { SearchFacetFiltersService } from '../../../services/search-facet-filters.service';
+import { SimpleChange } from '@angular/core';
+
+describe('SearchFacetChipTabbedComponent', () => {
+  let component: SearchFacetChipTabbedComponent;
+  let fixture: ComponentFixture<SearchFacetChipTabbedComponent>;
+  let queryBuilder: SearchQueryBuilderService;
+  let searchFacetService: SearchFacetFiltersService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+        imports: [
+            TranslateModule.forRoot(),
+            ContentTestingModule
+        ]
+    });
+    fixture = TestBed.createComponent(SearchFacetChipTabbedComponent);
+    component = fixture.componentInstance;
+    queryBuilder = TestBed.inject(SearchQueryBuilderService);
+    searchFacetService = TestBed.inject(SearchFacetFiltersService);
+    spyOn(queryBuilder, 'update').and.stub();
+
+    const facet1: FacetField = { type: 'field', label: 'field', field: 'field', buckets: new SearchFilterList() };
+    const facet2: FacetField = { type: 'field', label: 'field2', field: 'field2', buckets: new SearchFilterList() };
+
+    const tabbedFacet: TabbedFacetField = {
```

Comment:
this is redundant because you directly assign it to component.tabbedFacet in next line, you can remove that const

Reply:
Right, changed